models/quickbooks.yml

```
@@ -394,6 +413,12 @@ models:
         description: Balance of the account as of the first date of the period. 
       - name: period_ending_balance
         description: Sum of all transaction line adjusted_balances of the account for the period.
+      - name: period_net_converted_change
+        description: Account balance difference, converted with exchange rates applied if available, from the beginning and ending of the period.
+      - name: period_beginning_converted_change
+        description: Balance of the account, converted with exchange rates applied if available, as of the first date of the period. 
+      - name: period_ending_converted_change
```

The model code has the below name. I believe this is the appropriate name, if so we should change that here.

```suggestion
      - name: period_ending_converted_balance
```